reconstruction/cvt/src/main/java/org/jlab/rec/cvt/services/RecUtilities.java

```diff
@@ -867,9 +867,9 @@ static double getTrackKey(Seed s) {
     }
 
     Track recovTrkMisClusSearch(Seed seed, org.jlab.clas.tracking.trackrep.Helix hlx, double[][] cov, KFitter kf2, KFitter kf, int pid, 
-            List<Surface> surfaces, double xb, double yb, List<Cluster> SVTclusters, List<Cross> SVTcrosses, 
+            List<Surface> surfaces0, double xb, double yb, List<Cluster> SVTclusters, List<Cross> SVTcrosses, 
             Swim swimmer, double solenoidScale, double solenoidValue, Measurements measure) {
-        
+        List<Surface> surfaces = surfaces0;
```

Comment:
I agree with Raffaella. 
List, as a function parameter, is passed by reference. 
So it seems no difference before and after change.

Reply:
I agree with Raffaella.
List, as a function parameter, is passed as reference.
Meanwhile, assignment by "=" is also assigned as reference instead of value.
I do not see difference before and after change.

Reply:
https://github.com/JeffersonLab/clas12-offline-software/compare/CVTStatusFix2#diff-34fb52cfd7338bb3d130128b19e3b6cce553a78f97b1a7ee530e0eb309ea4d3fR930-R937
In block of "else", track status is set for "if(fittedTrack!=null)", how about cases for fittedTrack==null?

Reply:
https://github.com/JeffersonLab/clas12-offline-software/compare/CVTStatusFix2#diff-11dca748f6ec28aa8c86a243322b98bd6bd1edbf0a317153b6aca992c66f220bL551-R551
I can understand that the line is commented since the return value from the function surface.getEloss is not applied.
Does it mean energy loss is not applied for central tracking?

Reply:
We found out that the Loss needed to be turned off for these outer detectors.